linkerd/app/core/src/lib.rs

```diff
@@ -52,7 +52,6 @@ pub mod transport;
 pub use self::addr_match::{AddrMatch, IpMatch, NameMatch};
 
 pub const CANONICAL_DST_HEADER: &str = "l5d-dst-canonical";
-pub const DST_OVERRIDE_HEADER: &str = "l5d-dst-override";
 
 const DEFAULT_PORT: u16 = 80;
```

Comment:
looks like this constant is no longer used --- should we remove it?
```suggestion
```

Reply:
it's used below